Documentation/guides/port_relatedkernelconfigrations.rst

```diff
@@ -0,0 +1,65 @@
+The list of related kernel configurations
+=========================================
+
+The premise of this list : NuttX12.4.0, cxd56xx, non-SMP and Flat build.
+
+I found following kernel configurations by analyzing the boot sequence. I think it is good to consider them, although almost of them might be set as default.
+
++-----------------+-------------------------------+---------------------------------------------------------------------------------------------------------------------+
```

Comment:
could you make these tables smaller? in this state it is impossible to read these doc pages in terminal based IDE

Reply:
you can move links to the bottom of the table instead of pasting them directly into the table